sync/syncer.go

```diff
@@ -189,24 +195,28 @@ func (s *Syncer) lastTickedLayer() types.LayerID {
 }
 
 func (s *Syncer) Synchronise() {
-	for currentSyncLayer := s.lValidator.ValidatedLayer() + 1; currentSyncLayer <= s.lastTickedLayer(); currentSyncLayer++ {
-		lastTickedLayer := s.lastTickedLayer()
-		s.Info("syncing layer %v current consensus layer is %d", currentSyncLayer, lastTickedLayer)
-
-		if s.IsSynced() && currentSyncLayer == lastTickedLayer {
-			continue
-		}
-
+	defer s.syncFuncWg.Done()
+	currentSyncLayer := s.lValidator.ValidatedLayer() + 1
+	if s.IsSynced() {
 		lyr, err := s.GetLayer(types.LayerID(currentSyncLayer))
 		if err != nil {
-			s.Info("layer %v is not in the database", currentSyncLayer)
-			if lyr, err = s.getLayerFromNeighbors(currentSyncLayer); err != nil {
-				s.Info("could not get layer %v from neighbors %v", currentSyncLayer, err)
-				return
-			}
+			s.Error("err not nil err=%v", err)
```

Comment:
please something more descriptive this is a serious error 
something like "failed  getting layer even though IsSynced is true"

Reply:
Done